### ISSUE TITLE
Model connections while using Aurora database.

### ISSUE THREAD

**Issue by rajcrystaldelta** [aws-amplify/amplify-cli#4417]
>**Note**: If your question is regarding the AWS Amplify Console service, please log it in the 
[official AWS Amplify Console forum](https://forums.aws.amazon.com/forum.jspa?forumID=314&start=0)

**Which Category is your question related to?**
Model connections when Amplify uses Aurora as dataSource.

**Amplify CLI Version** 
4.18.1

**What AWS Services are you utilizing?**
Amplify, Appysync, Aurora Mysql

**Provide additional details e.g. code snippets**
Hi Team,
 I followed this [Amplify Doc](https://docs.amplify.aws/cli/graphql-transformer/relational) to add `Amazon Aurora serverless DB` to Amplify project. 

1. Added GrapgQL API by running `amplify add api`
![amplify_add_api](https://user-images.githubusercontent.com/32514653/83348361-632c2900-a349-11ea-999a-30aa7b5529af.png)

2. added aurora mysql serverless by `amplify api add-datasource` 
![add_graphql_datasource](https://user-images.githubusercontent.com/32514653/83348362-69baa080-a349-11ea-982f-e3f8922fa9dc.png)

My `Schema.graphql` looks like 
```
type Customers {
  id: Int!
  name: String!
  phone: String!
  email: String!
}

type Orders {
  id: Int!
  customerId: Int!
  orderDate: AWSDateTime
}

input CreateCustomersInput {
  id: Int!
  name: String!
  phone: String!
  email: String!
}

input CreateOrdersInput {
  id: Int!
  customerId: Int!
  orderDate: AWSDateTime
}

type Mutation {
  createCustomers(createCustomersInput: CreateCustomersInput!): Customers
  createOrders(createOrdersInput: CreateOrdersInput!): Orders
}

type Query {
  getCustomers(id: Int!): Customers
  listCustomerss: [Customers]
  getOrders(id: Int!): Orders
  listOrderss: [Orders]
}
```

Now all mutations and queries are working. Now I need to get a list of orders by customers like,

```
query ListCustomerWithOrders {
  id
  name
  order {
    id
    orderDate
  }
}
```

How I do the connections in the schema above. I guess `@connections` support only of we use `@model` but here am using Aurora serverless RDS?

**Comment by nikhname** [comment]
Hi @rajcrystaldelta, unfortunately RDS does not support any of the graphQL transformer directives at this time. You will need to model connections by creating the relationship in SQL. For example to model a connection between customers and order you could make the primary key of the Customers table the secondary key of the Orders table. You can take a look at. https://github.com/aws-amplify/amplify-cli/issues/3120 for more details

**Comment by cookmscott** [comment]
@nikhname , I don't understand your explanation that SQL must contain the relationship in order for this to work. The two tables do have foreign key constraints. Raj used the amplify docs which creates a customer and orders table with PKs and FKs for each: https://docs.amplify.aws/cli/graphql-transformer/relational

It is super unclear to me (and to many) how you query across tables in GraphQL if @connections and @model is not supported for Aurora. How do you query a customer and their orders IN THE EXAMPLE GIVEN IN THE DOCS? 

This has so many people stuck.. I've scoured the internet for solutions so any help here would be greatly appreciated.

**Comment by cdac901** [comment]
+1
